tests/p2p/test_connections.py

```diff
@@ -23,6 +23,6 @@ def test_manager_connections(self):
         endpoint = 'tcp://127.0.0.1:8005'
         manager.connections.connect_to(endpoint, use_ssl=True)
 
-        self.assertNotIn(endpoint, manager.connections.iter_not_ready_endpoints())
+        self.assertIn(endpoint, manager.connections.iter_not_ready_endpoints())
```

Comment:
Why have you inverted the logic?

Reply:
For the same reason described in https://github.com/HathorNetwork/hathor-core/pull/694#discussion_r1286423313

Reply:
Got it. Thanks!